src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -68,6 +69,8 @@ public static int MainCore(string workingDirectory, string[] args)
             }
 
             var console = new Common.Console();
+            Logger.Instance = console;
```

Comment:
This creates new instance on every run. Can be dangerous when running tests calling Program.Main directly in parallel.